app/scripts/views/settings/recovery_codes.js

```diff
@@ -127,7 +127,7 @@ const View = FormView.extend({
     this.recoveryCodesText = '';
     if (codes) {
       this.recoveryCodes = codes;
-      this.recoveryCodesText = this.recoveryCodes.join('\n');
+      this.recoveryCodesText = this.recoveryCodes.join(' \n\r');
```

Comment:
I think it needs to be `"\r\n"`, not the other way around

Reply:
Indeed it does